### ISSUE TITLE
Automatically regenerate IDE integration steps

### ISSUE THREAD

**Issue by grahamalama** [RobotPajamas/suspenders#125]
Initially posted this as a question in the Slack and was encouraged to convert it to an issue here.

The docs have a [clear section](https://www.pantsbuild.org/2.21/docs/using-pants/setting-up-an-ide) on how to set up some basic IDE integrations (with VSCode, in my case):

First party deps:
```
ROOTS=$(pants roots)
python3 -c "print('PYTHONPATH=./' + ':./'.join('''${ROOTS}'''.replace(' ', '\\ ').split('\n')) + ':\$PYTHONPATH')" > .env
```

Third party deps:
```
pants export --py-resolve-format=symlinked_immutable_virtualenv --resolve=python-default
```

It would be nice to rerun these steps automatically when either a source is changed (for first party deps) or a 3rd party dependency is changed.

**Comment by sureshjoshi** [comment]
Yep, this is a great idea - could be done pretty reasonably as part of some post-pants.toml change hooks that the plugin already needs to have.

Ideally could be back ported into pants.
